.github/workflows/ci.yml

```diff
@@ -134,4 +134,4 @@ jobs:
         with:
           workspaces: starknet-foundry
       - run: cargo lint
-        working-directory: ./starknet-foundry
+        working-directory: .
```

Comment:
I think we can remove the working-directory line completely then

Reply:
Sure, will do it in other PR though, I refuse to wait another 25 minutes to merge it xD

Reply:
I would not consider it as a good practice 😅